driver/gl/window.go

```diff
@@ -503,50 +593,104 @@ func keyToName(key glfw.Key) fyne.KeyName {
 		return desktop.KeySuper
 	case glfw.KeyMenu:
 		return desktop.KeyMenu
-
-	case glfw.KeyKPEnter:
-		return fyne.KeyEnter
 	}
 	return ""
 }
 
 func (w *window) keyPressed(viewport *glfw.Window, key glfw.Key, scancode int, action glfw.Action, mods glfw.ModifierKey) {
-	if w.canvas.Focused() == nil && w.canvas.onTypedKey == nil {
-		return
-	}
+
 	if action != glfw.Press { // ignore key up
+		// TODO: handle repeated key
```

Comment:
I don't think this is right - the keys to repeat are the runes.
Also can the space above be removed?

Reply:
It seems that the key repeat action is only available in this function. Anyway it is out of scope here :)
I'll create a new PR for this